src/token/api/src/canister/is20_transactions.rs

```diff
@@ -80,12 +85,14 @@ pub fn batch_transfer(
         .to_tokens128()
         .ok_or(TxError::AmountOverflow)?;
 
-    if balances.balance_of(&from) < (total_value + total_fee).ok_or(TxError::AmountOverflow)? {
+    if balances.balance_of(from) < (total_value + total_fee).ok_or(TxError::AmountOverflow)? {
         return Err(TxError::InsufficientBalance);
     }
 
     {
-        for (to, value) in transfers.clone() {
+        for x in transfers.clone() {
```

Comment:
easy enough to read today, but perhaps not in the future.
change `x` to `transfer`
```suggestion
        for x in &transfers {
```